ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml

```diff
@@ -2614,6 +2614,13 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-vsphere-ipi-template
+- as: vsphere-ipi-usertags-f28
```

Comment:
Maybe we can merge the coverage of `usertags` to other existing ci jobs to save some slice quota.

Reply:
sounds good.